src/core/layout/navbar/responsive-drawer/drawer/drawer-component.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useCallback } from 'react'
+import { Drawer } from '@material-ui/core'
+
+import { useContext } from 'core/layout/navbar/context'
+import { useStyles } from 'core/layout/navbar/responsive-drawer/drawer/style'
+import { DrawerContent } from 'core/layout/navbar/responsive-drawer/drawer/drawer-content'
+
+const useOnClose = () => {
```

Comment:
Let's extract these hooks to separate files.

Reply:
Done